src/main/java/io/edpn/backend/messageprocessorlib/application/dto/eddn/journal/ScanMessage.java

```diff
@@ -0,0 +1,103 @@
+package io.edpn.backend.messageprocessorlib.application.dto.eddn.journal;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.edpn.backend.messageprocessorlib.application.dto.eddn.EddnMessage;
+import io.edpn.backend.messageprocessorlib.application.dto.eddn.TimestampedEddnMessage;
+import io.edpn.backend.util.TimestampConverter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+public interface ScanMessage {
+    
+    String EVENT = "JournalScan";
+    
+    record V1 (
+        @JsonProperty("$schemaRef") String schemaRef,
+        @JsonProperty("header") EddnMessage.Header header,
+        @JsonProperty("message") V1.Payload message) implements TimestampedEddnMessage<EddnMessage.Payload> {
+        
+        @Override
+        public LocalDateTime messageTimeStamp() {
+            return TimestampConverter.convertToLocalDateTime(message.timestamp);
+        }
+        
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        public record Payload(
+                //Star or Body
+                @JsonProperty("DistanceFromArrivalLS") Double arrivalDistance,
+                @JsonProperty("AxialTilt") Double axialTilt,
+                @JsonProperty("WasDiscovered") Boolean discovered,
+                @JsonProperty("WasMapped") Boolean mapped,
+                @JsonProperty("BodyName") String name,
+                @JsonProperty("BodyID") Long localID,
```

Comment:
From the few examples I looked up, seems like `localId` would be more consistent naming.